.github/workflows/build_and_test.yml

```diff
@@ -2,9 +2,11 @@ name: Build and test
 
 on:
   push:
-    branches: [ python-backend ]
```

Comment:
Just to be sure: the idea is that the automated commit will also work if not going through the PR process?

Reply:
Yeah, auto-generation should be executed for every push I think.

* If it's a PR from another fork, I think the action should work in that fork, so we still receive a commit.
* If it's our PR, the action will work in our branch.
* If it's pushed just in the default branch without a PR, the automation still happens.